.eslintrc

```diff
@@ -21,6 +21,7 @@
       }
     }
   },
+  "ignorePatterns": ["packages/**/*.d.ts"],
```

Comment:
We won't need this anymore because of the `eslintignore` change in #2857 right?

Reply:
yep! I'll remove that!!